spec/support/planned_subjects_test_helper.rb

```diff
@@ -22,7 +22,11 @@ def assert_available_subject(subject_name)
   end
 
   def within_planned_subjects(&block)
-    within(:xpath, "//h3[contains(text(), 'Materias planeadas')]/following-sibling::*[1]", &block)
+    within(:xpath, "//div[@id='planned-subjects']/following-sibling::*[1]", &block)
```

Comment:
Evitaría poner un `id` al elemento en el html solo por los tests. Creo que esto debería andar:

```suggestion
    within(:xpath, "//div[h3[contains(text(), 'Materias planeadas')]]/following-sibling::*[1]", &block)
```

Reply:
done